server/spinwick.go

```diff
@@ -349,7 +543,19 @@ func (s *Server) updateSpinWick(pr *model.PullRequest, withLicense bool) *spinwi
 }
 
 func (s *Server) handleDestroySpinWick(pr *model.PullRequest) {
-	request := s.destroySpinWick(pr)
+	request := &spinwick.Request{
+		InstallationID: "n/a",
+		Error:          nil,
+		ReportError:    false,
+		Aborted:        false,
+	}
+
+	if pr.RepoName == "mattermost-webapp" || pr.RepoName == "mattermost-server" {
```

Comment:
we can have other repos that spin MM servers not only "mattermost-webapp" or "mattermost-server"

maybe just check if is from "customer-web-server" then call destroyKubeSpinWick otherwise go straight to destroySpinWick